dbms/src/Flash/tests/gtest_split_tasks.cpp

```diff
@@ -95,15 +95,15 @@ try
 
     size_t task_size = tasks.size();
     std::vector<String> executors = {
-        "exchange_sender_6 | type:Hash, {<0, String>}\n"
-        " table_scan_1 | {<0, String>}",
+        "exchange_sender_6 | type:Hash, {<0, String>, <1, String>}\n"
+        " table_scan_1 | {<0, String>, <1, String>}",
         "exchange_sender_5 | type:Hash, {<0, String>, <1, String>}\n"
         " table_scan_0 | {<0, String>, <1, String>}",
         "exchange_sender_4 | type:PassThrough, {<0, String>, <1, String>, <2, String>}\n"
```

Comment:
It looks like the output of `exchange_sender_4` is wrong, it should be 4 columns

Reply:
> It looks like the output of `exchange_sender_4` is wrong, it should be 4 columns

Ops, also solved in another pr, I will move the solution to this pr.